jupyter-book/preprocessing_visualization/preprocessing.ipynb

```diff
@@ -0,0 +1,653 @@
+{
```

Comment:
<p>What you wrote is great, really, but I think that a little bit of biological context is missing. What is a bad quality cell and why are we filtering them (from a biological view)?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/extended-single-cell-best-practices/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>